README.md

```diff
@@ -1,76 +1,154 @@
 # Fractal ID API Documentation
 
 Fractal ID is an online service for identity provisioning and verification.
-This service implements the OAuth2 protocol for user authentication,
-authorization and resource retrieval.
+This service implements the [OAuth2](https://oauth.net/2/) protocol for user
+authentication, authorization and resource retrieval.
 
-## Setup
+## Getting started
 
-The integration of Fractal ID API is currently available for select partners.
-Setup involves:
+Integration of Fractal ID API is currently available upon request.
 
-1. the partner providing Fractal with:
-   1. the target authentication redirection endpoint (`redirect_uri`);
+To get started, we need you to provide us with:
+   1. the target authorization redirection endpoint, `redirect_uri`
+      (`https://example.com/oauth/callback`);
+   1. the application homepage URL;
    1. the application display name;
    1. the application display logo (in square format);
-1. Fractal providing the partner with an API application ID and secret (`client_id` and `client_secret`).
+   1. optionally, the webhook callback URL (`https://example.com/webhook`);
 
-## User authentication and authorization
+We will then provide you with an API application ID and secret (`CLIENT_ID` and
+`CLIENT_SECRET`). If you are using Webhooks, we will also send you the webhook
+secret token (`WEBHOOK_SECRET`).
 
-Fractal ID implements the [authorization code grant
-flow](https://tools.ietf.org/html/rfc6749#section-1.3.1) of the [OAuth2
-standard](https://tools.ietf.org/html/rfc6749).
+Note: As a security measure, all URLs [MUST be secured using
+HTTPS](https://tools.ietf.org/html/rfc6749#section-3.1.2.1), except when using
+`localhost` addresses.
+
+We have staging and production environments available. throughout this
+document, we will use the following placeholders. You should use the domains
+that match the environment you are using.
+
+| domain            | staging                  | production          |
+| ----------------- | ------------------------ | ------------------- |
+| `FRONTEND_DOMAIN` | next.fractal.id          | fractal.id          |
+| `AUTH_DOMAIN`     | auth.next.fractal.id     | auth.fractal.id     |
+| `RESOURCE_DOMAIN` | resource.next.fractal.id | resource.fractal.id |
+| `VERIFIER_DOMAIN` | verifier.next.fractal.id | verifier.fractal.id |
+
+
+## KYC levels
+
+Fractal ID supports different levels of verification. Additionally, each level
+may be extended by a number of addons.
```

Comment:
```suggestion
may be extended by a number of addons. This will impact the data the user is requested to share and consequently their journey in Fractal ID.
```